Assets/CritterController.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class CritterController : MonoBehaviour
+{
+    public float consumed, hydrated;
+    public float consumptionLostPerSecond;
+    public float hydrationLostPerSecond;
+    public float force;
+
+    public GameObject target;
+    private Rigidbody rb;
+
+    void Start()
+    {
+        gameObject.GetComponent<Renderer>().material.color = Color.red;
```

Comment:
Maybe this weekend somebody can make a fun material and pop it on there

Reply:
Big argument against this is that the code is no longer in control of the color, but: even if we want to dynamically change materials later (eg. turn red when you get hurt) the code can swap out materials we make too, not just colors

Reply:
Yeah, that sounds good to me. I partly did it that way so we could change the color with the code in the future. But honestly, it was mainly just me being lazy instead of creating a new material.